src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.product-card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: center;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.product-card__value {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.product-card__cash {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.product-card__button {
+  display: flex;
+  height: 40px;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+  line-height: 18px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid transparent;
+  transition: 0.3s linear;
```

Comment:
Specify the transition properties you change on hover

Reply:
Can you explain, please? I don't quite understand.